packages/grafana-ui/src/components/Modal/Modal.story.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import { Modal } from './Modal';
+
+const RefreshSelectStories = storiesOf('UI/Modal', module);
+
+RefreshSelectStories.addDecorator(withCenteredStory);
+
+const LazyComponent = React.lazy(() => import('./LazyExample'));
```

Comment:
Can we simulate the laziness instead of actually importing it? Not very happy with the LazyExample.tsx file. Maybe put it in `mocks` dir?